unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/Environment.cs

```diff
@@ -52,6 +54,7 @@ public void Initialize(IMessageProcessHandler messageHandler, ISceneHandler scen
             worldBlockersController = WorldBlockersController.CreateWithDefaultDependencies(sceneHandler, DCLCharacterController.i.characterPosition);
             parcelScenesCleaner.Start();
             cullingController.Start();
+            worldState = new WorldState();
```

Comment:
Why an instance of WorldState is created both inside Environment constructor and inside Initialize method?

Reply:
To reset its state